02-ServiceRegistration/src/PrismSample/Views/ViewAPage.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace PrismSample.Views
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
```

Comment:
remove this we already have it at the assembly level... be sure to remove unused namespaces as well ;)

Reply:
Done with the latest commit :)